src/app/core/interceptors/http.interceptor.ts

```diff
@@ -66,6 +68,12 @@ export const httpInterceptor: HttpInterceptorFn = (req, next) => {
           router.navigate(['/logout']);
         }
 
+        if ((isRefreshRequest || isLoginRequest)  && error.status === 500) {
+          setTimeout(() => {
+            window.location.reload();
+          }, RELOAD_TIMEOUT_MS);
+        }
+
```

Comment:
Would you mind explaining why the timeout and the length of it mostly for me to understand